### PR TITLE
Bump version to 5.0.0-alpha5

### PR DESCRIPTION
**Pull request by suyograo**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -21,94 +21,93 @@ gem "gems", "~> 0.8.3", :group => :build
...
+gem "logstash-output-rabbitmq"
+gem "logstash-output-redis"
+gem "logstash-output-s3"
+gem "logstash-output-sns"
+gem "logstash-output-sqs"
+gem "logstash-output-statsd"
+gem "logstash-output-stdout"
+gem "logstash-output-tcp"
+gem "logstash-output-udp"
+gem "logstash-output-webhdfs"
```
Comment (ph) [inline Gemfile]: Are we removing all the version constrains when we release an official version, I though we were keeping them?

File: versions.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
-logstash: 5.0.0-alpha4
-logstash-core: 5.0.0-alpha4
-logstash-core-event: 5.0.0-alpha4
-logstash-core-event-java: 5.0.0-alpha4
-logstash-core-plugin-api: 2.1.9
+logstash: 5.0.0-alpha5
+logstash-core: 5.0.0-alpha5
+logstash-core-event: 5.0.0-alpha5
+logstash-core-event-java: 5.0.0-alpha5
+logstash-core-plugin-api: 2.1.10
```
Comment (ph) [inline versions.yml]: <3

**Comment by ph** [comment]
minor comments concerning the version constrains in the Gemfile.

File: Gemfile (hunk 1 above)
Comment (suyograo) [inline Gemfile]: I think we should do that for a GA, not a pre-release.. its too early now, and plugins are getting upgraded

Comment (ph) [inline Gemfile]: in slack, we agreed to only do it on GA

**Comment by ph** [comment]
LGTM, comment was addressed on slack.

**Pull request merged**